client/src/use/useTrackStore.ts

```diff
@@ -75,17 +80,20 @@ export default function useTrackStore({ markChangesPending }: UseTrackStoreParam
     markChangesPending({ action: 'upsert', track });
   }
 
-  function insertTrack(track: Track, afterId?: TrackId) {
+  function insertTrack(track: Track, args?: InsertArgs) {
     track.bus.$on('notify', onChange);
     trackMap.set(track.trackId, track);
     intervalTree.insert([track.begin, track.end], track.trackId.toString());
-    if (afterId) {
+    if (args && args.afterId) {
       /* Insert specifically after another trackId */
-      const insertIndex = trackIds.value.indexOf(afterId) + 1;
+      const insertIndex = trackIds.value.indexOf(args.afterId) + 1;
       trackIds.value.splice(insertIndex, 0, track.trackId);
     } else {
       trackIds.value.push(track.trackId);
     }
+    if (!args || !args.imported) {
```

Comment:
Maybe this reads better?

```suggestion
    if (!args?.imported) {
```